app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/BankAccountController.scala

```diff
@@ -34,14 +36,19 @@ class BankAccountController @Inject() (
   val sessionStore: SessionCache,
   val errorHandler: ErrorHandler,
   cc: MessagesControllerComponents,
+  checkBankAccountDetailsPage: views.html.claims.bank_account_details,
   val config: Configuration
-) extends FrontendController(cc)
+)(implicit viewConfig: ViewConfig)
+    extends FrontendController(cc)
     with WithAuthAndSessionDataAction
     with Logging
     with SessionUpdates {
 
-  def checkBankAccountDetails(): Action[AnyContent] = Action {
-    Ok("bank account details - to be implemented")
+  def checkBankAccountDetails(): Action[AnyContent] = authenticatedActionWithSessionData { implicit request =>
+    Ok(checkBankAccountDetailsPage(MaskedBankAccount("Shiva", "45-45-45", "123456")))
```

Comment:
you've left some dummy data here

Reply:
I had to leave the dummy data here so that the page would render  - I can take it out, but that would mean modifying the html page to take out the form data